tests/FSharp.Compiler.Service.Tests/ProjectAnalysisTests.fs

```diff
@@ -4645,7 +4645,7 @@ let callToOverload = B(5).Overload(4)
     let args = mkProjectCommandLineArgs (dllName, [])
 
 [<Theory>]
```

Comment:
Also, with transparent compiler CI branch we can phase out these theories and use `CompilerAssertHelpers.UseTransparentCompiler` instead.

Reply:
I converted some of those now, but it will require a more thorough search.